### PR TITLE
Create es-cl.js

### PR DESCRIPTION
**Pull request by blogcraft**
locale for Chile es-cl

Date separator is  "-" instead of "/" used in Standard Spanish.
Hour format is 24 hour.

### CODE REVIEW

File: src/locale/es-cl.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+//! author : Gerardo Buenrostro : https://github.com/blogcraft
+
+import moment from '../moment';
+
+var monthsShortDot = 'ene._feb._mar._abr._may._jun._jul._ago._sep._oct._nov._dic.'.split('_'),
+    monthsShort = 'ene_feb_mar_abr_may_jun_jul_ago_sep_oct_nov_dic'.split('_');
+
+export default moment.defineLocale('es', {
+    months : 'enero_febrero_marzo_abril_mayo_junio_julio_agosto_septiembre_octubre_noviembre_diciembre'.split('_'),
+    monthsShort : function (m, format) {
```
Comment (ichernev) [inline src/locale/es-cl.js]: Take a look at the `be` locale for an example of declarative standalone/format.

**Comment by ichernev** [comment]
@blogcraft also you'd need to add tests. Just copy another test file and modify it to pass.

**Comment by ichernev** [comment]
@blogcraft to add tests just copy another file from `src/test/locale/es.js` and modify so it works.
You'd also need to sign the CLA (new requirement).

**Comment by mj1856** [comment]
@blogcraft - Please [sign the CLA](https://cla.js.foundation/moment/moment?pullRequest=3510) and also add unit tests to this PR.  Thanks.

**Comment by maggiepint** [comment]
Closing. Feel free to re-open if tests are added

**Pull request closed without merging**